packages/hint-compat-api/tests/css-next.ts

```diff
@@ -239,6 +239,20 @@ const mixedFeaturedCompatibility: HintTest[] = [
 
 hintRunner.testHint(hintPath, mixedFeaturedCompatibility, { browserslist: ['firefox 63', 'edge 18'], parsers: ['css']});
 
+const notSupportedFeaturesShouldNotSeparatelyLog: HintTest[] = [
+    {
+        name: 'Features not supported should not separately log the feature and value.',
+        reports: [
+            { message: 'appearance prefixed with -webkit- is not supported on ie browser.', position: { column: 4, line: 1 }},
+            { message: 'appearance prefixed with -moz- is not supported on ie browser.', position: { column: 4, line: 2 }},
+            { message: 'appearance is not supported on ie browser.', position: { column: 4, line: 3 }}
```

Comment:
@borgitas21 Looks like this is still happening and the `compat-api/css` hint is logging these same reports in addition to the `compat-api/css-next` hint. As this is not a deprecated feature, only `compat-api/css-next` should be reporting it.

Here's the log I get in the "Problems" panel when opening the `hint-compat-api/tests/fixtures/css/appearance.css` in VS Code using our extension:
```
[webhint] appearance prefixed with -webkit- is not supported by ie. (compat-api/css)
[webhint] appearance prefixed with -webkit- is not supported by ie. (compat-api/css-next)
[webhint] appearance prefixed with -moz- is not supported by ie. (compat-api/css)
[webhint] appearance prefixed with -moz- is not supported by ie. (compat-api/css-next)
[webhint] appearance is not supported by ie. (compat-api/css)
[webhint] appearance is not supported by ie. (compat-api/css-next)
```

The expected result is just one report:
```
[webhint] appearance is not supported by ie. (compat-api/css-next)
```

FWIW I'm okay tracking this as a separate issue for a follow-up PR if we want to get this PR wrapped up and merged. That said, I do think we should fix this before turning the hint on-by-default.

Reply:
Likely related, missing support for the `integrity` attribute is getting flagged by both `compat-api/html` and `compat-api/html-next` (only the latter should report this).

Note that the list of browsers differs between these as well, which seems like a separate bug.

Actual:
```
[webhint] integrity attribute of the link element is not supported by edge, ie, safari, safari_ios. (compat-api/html)
[webhint] integrity attribute of the link element is not supported by edge, ie, safari, safari_ios, samsunginternet_android 4, webview_android 4.4.3-4.4.4. (compat-api/html-next)
```

Expected:
```
[webhint] integrity attribute of the link element is not supported by edge, ie, safari, safari_ios, samsunginternet_android 4, webview_android 4.4.3-4.4.4. (compat-api/html-next)
```

Reply:
@antross I think this is getting in conflict with:

> Yes, I think we must "merge" the reports to provide a single report about deprecation coming from compat-api/html.
>
> To be clear, at an implementation level I don't really see this as a "merge", but rather a change in conditions to treat browsers that never implemented a deprecated feature as part of the "deprecated" set (compat-api/html, compat-api/css) instead of the "not broadly supported" set (compat-api/html-next, compat-api/css-next).

Previously, we moved the "never implemented features" to the deprecated hint. If I understood correctly, now we want to move it to the next hint. Anyway, I found an issue because we were still handling never implemented features by the next hint in some scenarios. However we still get two different reports. Am I missing something? Could you bring some light to the subject?

```
[webhint] integrity attribute of the link element is not supported by edge, ie, safari, safari_ios. (compat-api/html)
[webhint] integrity attribute of the link element is not supported by samsunginternet_android 4, webview_android 4. (compat-api/html-next)
```

Reply:
Neat 👏 . Thanks! Please, let me know your feedback after my last change.

About the previous issue, I'm aware of it, but yes, please, let's tackle it in an specific issue since this is not part of this development and I don't want to delay more this PR.